meaningfulconsent/templates/main/participant_sessions.html

```diff
@@ -49,6 +49,7 @@
                     <td>
                     <div class="pull-right">
                         <form action="/participant/login/" method="post" style="display: inline-block">
+                            <input type="hidden" name="csrfmiddlewaretoken" value="<%=csrfmiddlewaretoken%>">
```

Comment:
I always feel weird about mixing django template tags into client-side templating. But, it's a good point. It's probably better to use the template tag in this case.